cmd/kyma/alpha/deploy/cmd.go

```diff
@@ -145,8 +145,14 @@ func (cmd *command) Run() error {
 	cmd.duration = time.Since(start)
 
 	// import certificate
-	if err := cmd.importCertificate(); err != nil {
-		return err
+	qImportCertsStep := cmd.NewStep("Install Kyma certificate locally")
+	if !cmd.avoidUserInteraction() && qImportCertsStep.PromptYesNo("Should the Kyma certificate be installed locally") {
+		if err := cmd.importCertificate(); err != nil {
+			return err
+		}
+		qImportCertsStep.Success()
+	} else {
+		qImportCertsStep.Failure()
```

Comment:
Why should the step fail if the user choses not to import the certificate or we are in non interactive mode?
Failing means something went wrong.